addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -0,0 +1,174 @@
+        FROM res_company rc
+        WHERE act.%s = rc.id AND act.cash_account_code_prefix IS NULL
+        """, (AsIs(openupgrade.get_legacy_name('company_id')), ),
+    )
+    # if company_id was not filled:
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET bank_account_code_prefix = 'OUB'
+        WHERE act.bank_account_code_prefix IS NULL
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET cash_account_code_prefix = 'OUB'
+        WHERE act.cash_account_code_prefix IS NULL
+        """
+    )
+    # transfer_account_code_prefix:
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET transfer_account_code_prefix = trim(leading '0' from aat.code)
+        FROM account_account_template aat
+        WHERE act.%s = aat.id
+        """, (AsIs(openupgrade.get_legacy_name('transfer_account_id')), ),
+    )
+
+
+def update_res_company_account_setup_steps_states(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_bank_data_state = 'done'
+        WHERE account_setup_bank_data_done = TRUE
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_fy_data_state = 'done'
+        WHERE account_setup_fy_data_done = TRUE
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_coa_state = 'done'
+        WHERE account_setup_coa_done = TRUE
+        """
+    )
+
+
+def fill_account_journal_alias_id(env):
+    journal_model = env["account.journal"]
+    journals = journal_model.with_context(
+        active_test=False).search([('type', '=', 'purchase')])
+    for journal in journals:
+        journal._update_mail_alias({})
+
+
+def fill_account_move_reverse_entry_id(env):
+    def fill_account_move_reverse_date(origin_move, reversal):
+        reverser = env['res.users'].browse(reversal.create_uid)
+        date = None
+        if origin_move.reverse_date and (
+                not reverser.company_id.period_lock_date
+                or origin_move.reverse_date >
+                reverser.company_id.period_lock_date):
+            date = reversal.reverse_date
+        reversal.reverse_date = date
+
+    all_moves = env['account.move'].search([])
+    if openupgrade.table_exists(env.cr, 'account_move_reverse'):
+        reversal_moves = all_moves.filtered(lambda m: m.reverse_entry_id).\
+            mapped('reverse_entry_id')
+        for move in reversal_moves:
+            origin = all_moves.filtered(
+                lambda m: m.reverse_entry_id == move.id)
+            fill_account_move_reverse_date(origin, move)
+    else:
+        reversal_moves = all_moves.filtered(
+            lambda m: m.ref and m.ref.startswith(_('reversal of: ')))
```

Comment:
This is not valid. Imagine you have users doing reversals in several languages. Don't you have any other way of finding this?

Reply:
I don't know of other way. Could you help me to provide a way? 😰

I hate when things are using translations. 😖 In `stock`module I also had the same problem.